sys/shell/shell.c

```diff
@@ -129,6 +129,38 @@ static shell_command_handler_t find_handler(
     return handler;
 }
 
+static void print_commands_json(const shell_command_t *cmd_list)
```

Comment:
So the JSON output itself is not sorted. I guess the rationale here is that a machine readable format can be more easily sorted later on than the human-readable one?